types/paradisebotsapi.js/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for paradisebotsapi.js 1.0
```

Comment:
```suggestion
// Type definitions for paradisebotsapi.js 1.10
```
If this is for 1.10.0, this should be changed